website/_docs31/tutorial/setup_systemcube.cn.md

```diff
@@ -118,6 +118,16 @@ sh ${KYLIN_HOME}/bin/kylin.sh org.apache.kylin.tool.job.CubeBuildingCLI --cube $
 
 {% endhighlight %}
 
+## 自动创建系统cube
```

Comment:
Please try to modify the content of `SCSinkTools.json` content for us. Thank you in advance.